vendor/engines/secure_rooms/app/services/secure_rooms/access_manager.rb

```diff
@@ -0,0 +1,22 @@
+module SecureRooms
+
+  class AccessManager
+
+    def self.process(verdict)
```

Comment:
What I was trying to suggest by improving the api is to leave everything that was there alone operating as primarily instance methods, but have a class method for a simpler interface. That gives you the advantage of being able to build out the instance methods without needing to pass around instance variables, but also have a cleaner interface. So something like:

```ruby
  def self.process(verdict)
    new(verdict).process
 end
 
 def process
   Event.create!(
     occurred_at: Time.current,
     card_reader: verdict.card_reader,
     user: verdict.user,
     # TODO: store a useful way of tracking the outcome
     outcome: verdict,
   )
 end
 ```